environments.json

```diff
@@ -1011,21 +1011,29 @@
     "obsolete": false
   },
   "firefox62": {
-    "full": "Firefox 62 Beta",
+    "full": "Firefox 62",
     "family": "SpiderMonkey",
-    "short": "FF 62 Beta",
+    "short": "FF 62",
     "release": "2018-09-05",
     "retire": "2018-10-23",
-    "unstable": true
+    "obsolete": false
   },
   "firefox63": {
-    "full": "Firefox 63 Nightly",
+    "full": "Firefox 63 Beta",
     "family": "SpiderMonkey",
-    "short": "FF 63 Nightly",
+    "short": "FF 63 Beta",
     "release": "2018-10-23",
     "retire": "2018-12-11",
     "unstable": true
   },
+  "firefox64": {
+    "full": "Firefox 64 Nightly",
+    "family": "SpiderMonkey",
+    "short": "FF 63 Nightly",
```

Comment:
I didn't notice this before the merge, but it needs to be fixed: it should be "FF **63** Nightly".